apps/prairielearn/src/middlewares/authzCourseOrInstance.ts

```diff
@@ -162,6 +163,9 @@ export async function authzCourseOrInstance(req: Request, res: Response) {
   // The side nav is expanded by default.
   res.locals.side_nav_expanded = req.session?.side_nav_expanded ?? true;
 
+  // Graded submissions are skipped by default.
+  res.locals.skip_graded_submissions = req.session?.skip_graded_submissions ?? true;
```

Comment:
This is only used in a few places. Rather than tacking it onto `res.locals` for every page, can we instead read directly from `req.session.skip_graded_submissions` wherever it's needed?